torch/csrc/autograd/python_function.cpp

```diff
@@ -743,6 +743,25 @@ PyObject *THPFunction_saved_variables(THPFunction *self, void *_unused)
   END_HANDLE_TH_ERRORS
 }
 
+PyObject *THPFunction_raw_saved_tensors(THPFunction *self, void *_unused)
+{
+  HANDLE_TH_ERRORS
```

Comment:
Maybe refactor this and unpack_saved_variables?

Reply:
The function I meant to refactor was ihttps://github.com/pytorch/pytorch/blob/7e619b95880cdec72d1353c9c99dedbc2575ad36/torch/csrc/autograd/python_function.cpp#L689

(they already slightly diverged, but I don't have strong preference in favor of refactoring)